bitacross-worker/app-libs/parentchain-interface/src/indirect_calls/litentry/scheduled_enclave.rs

```diff
@@ -28,7 +28,7 @@ use log::*;
 #[derive(Debug, Clone, Encode, Decode, Eq, PartialEq)]
 pub struct SetScheduledEnclaveArgs {
 	worker_type: WorkerType,
-	sbn: codec::Compact<SidechainBlockNumber>,
```

Comment:
Hey @Kailai-Wang I'm just curious on what the bug was and how it was discovered, does it affect other places where we are using `compact` encoding.

Reply:
Hi @felixfaisal - it's caused by parameter type mismatch: int `teebag.set_scheduled_enclave`, the `sbn` isn't compact encoded (so just plain `u64`), so you will get a decode error here without this fix.

It's easily reproducible when you try to set or remove scheduled enclave via indirect call.

Reply:
I think in `teerex` we are/were using `compact` - https://github.com/litentry/litentry-parachain/blob/3548f09b7ff80c2f8f9579d0c100cdff0846220e/pallets/teerex/src/lib.rs#L501 

It goes back to this issue - https://github.com/litentry/litentry-parachain/issues/1433 because of which we made the extrinsic parameter compact, and now we are kind of going back. Should we instead update `teebag` to use compact, or do you think there is no significant advantage in using `compact` encoding for our usecase currently?

Reply:
The latter :) compact encoding a u64 won't save us a lot but it's quite error-prone. We had the case where we forgot this compact encoding and had problems. It caused some trouble in ts-test.

That said maybe we should simply use plain encoding